pykafka/rdkafka/_rd_kafkamodule.c

```diff
@@ -0,0 +1,448 @@
+#include <Python.h>
+#include <structseq.h>
+
+#include <errno.h>
+
+#include <librdkafka/rdkafka.h>
+
+
+/**
+ * Exception types
+ */
+
+static PyObject *ConsumerStoppedException;
+static PyObject *PyRdKafkaError;
+
+
+static void
```

Comment:
Is separating the lines in this way the preferred C style? Seems like it would make more sense to insert linebreaks between arguments, but what do I know about C conventions?